nixos/tests/cagebreak.nix

```diff
@@ -56,6 +56,7 @@ in
       '';
     };
 
+
```

Comment:
Seems like this line was introduced on accident? (not a blocker but might make sense to remove)